FGB-insite-team-task/src/app/service/fgb-detail.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from "@angular/common/http";
+import {Game} from '../fgb-models/game.model';
+import { Season } from '../fgb-models/season.model';
+import { Product } from '../fgb-models/product.model';
+import { Attendance } from '../fgb-models/attendance.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FgbDetailService {
+  gameList:Game[];
+  seasonList:Season[];
+  productList:Product[];
+  attendanceList:Attendance[];
+  gameNumber:number;
+  seasonNumber:number;
+  productNumber:string;
```

Comment:
These properties should really be held in the component, or if you want to abstract them away from the component then you should have a separate store class for doing that. This type of service should be used only to get data from the API and deliver it to the calling class. Check out how it's done [here](https://angular.io/guide/http#requesting-data-from-a-server) for more detailed info.